config/Config-build.in

```diff
@@ -427,7 +427,7 @@ menu "Global build settings"
 		bool "Enable SECCOMP"
 		select KERNEL_SECCOMP
 		select PACKAGE_procd-seccomp
-		depends on (aarch64 || arm || armeb || mips || mipsel || mips64 || mips64el || i386 || powerpc || x86_64)
+		depends on (aarch64 || arm || armeb || mips || mipsel || mips64 || mips64el || i386 || powerpc || powerpc64 || x86_64)
```

Comment:
`loongarch64` should support this too.
@hackpascal  Could you please try this.

Reply:
@hauke if you look into https://github.com/openwrt/openwrt/pull/17003, I've already tried it, because those architectures/kernels support `seccomp`.

What is missing is support in `procd-seccomp`, so it wont compile/work. I don't have board for any of those architectures, thus I can't test it, so I don't plan adding that support myself. 

Additionally this is just a workaround, I've found out, that there is no reason, that `base-files`  pulls in `procd-seccomp` dependency on those architectures, because `CONFIG_SECCOMP=y` is *NOT*  set and `procd-seccomp` have `DEPENDS:=@SECCOMP`. So this needs more investigation, maybe @nbd168 will have an idea?

Reply:
This is rather weird, base-files is adding `procd-seccomp` to the depends list somehow

Reply:
Do we even need to select `procd-seccomp` via base-files since enabling SECCOMP selects it?
```
config SECCOMP
		bool "Enable SECCOMP"
		select KERNEL_SECCOMP
		select PACKAGE_procd-seccomp
```

Reply:
> `loongarch64` should support this too.
> @hackpascal Could you please try this.

I'll try this later

Reply:
@hauke I've tested and it works as expected for loongarch64.
PR: https://github.com/openwrt/openwrt/pull/17335
patch for procd: https://patchwork.ozlabs.org/project/openwrt/patch/20241222102812.773510-1-hackpascal@gmail.com/